Source/Core/Common/Debug/OSThread.cpp

```diff
@@ -0,0 +1,204 @@
+// Copyright 2020 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "Common/Debug/OSThread.h"
+
+#include <algorithm>
+#include "fmt/format.h"
```

Comment:
Sorry for missing this last time, but this should be

```suggestion
#include <fmt/format.h>
```